src/EditorFeatures/Core.Cocoa/Options/VisualStudioMacOptionsStorage.cs

```diff
@@ -64,14 +64,14 @@ public string GetName(string? language)
         {"dotnet_sort_system_directives_first", new PropertyName("TextEditor.%LANGUAGE%.Specific.PlaceSystemNamespaceFirst")},
         {"dotnet_separate_import_directive_groups", new PropertyName("TextEditor.%LANGUAGE%.Specific.SeparateImportDirectiveGroups")},
         {"QuickInfoOptions_ShowRemarksInQuickInfo", new PropertyName("TextEditor.%LANGUAGE%.Specific.ShowRemarks")},
-        {"FeatureOnOffOptions_AddImportsOnPaste", new PropertyName("TextEditor.%LANGUAGE%.Specific.AddImportsOnPaste2")},
+        {"dotnet_add_import_on_paste_options_add_imports_on_paste", new PropertyName("TextEditor.%LANGUAGE%.Specific.AddImportsOnPaste2")},
```

Comment:
I realize it's not always possible but should this just be grabbing the member and using a property off of it rather than duplicating strings?

Reply:
Duplicating string is ok.

Reply:
In this case, I removed the dup strings here